### DISCUSSION TITLE
Encountering issues loading tetr.io on Linux+webkitgtk

### DISCUSSION THREAD

**Discussion by huantianad** [tauri-apps/tauri#8659] [Q&A]
Hello, I'm attempting to make a simple desktop app wrapper for the site https://tetr.io. However, loading the site in a tauri results in some strange graphical issues, as well as a critical issue of the game not loading at all 😅. I'm unable to reproduce this issue in Epiphany, so I'm not sure where the issue lies. Any help would be appreciated! I've tried this on stable tauri as well as `2.0.0-alpha.21`.

Epiphany: (how it should look)
![image](https://github.com/tauri-apps/tauri/assets/20760920/506d3fb6-6b3c-442c-b32b-ef378a5667c5)

Tauri: (notice the issues with the CSS on the title bar and player counters. The game also is unable to finish loading, the Loading SFX that does not complete in the bottom right hand corner.
![Screenshot_2024-01-22_22:54:59](https://github.com/tauri-apps/tauri/assets/20760920/1a4408cd-764a-487f-bf11-4436e7323aa4)

**Comment by FabianLars** [comment]
Are there any warnings in the terminal or the devtools console?

**Reply by huantianad** [reply]
I did check dev tools, but I completely missed the gstreamer missing elements errors in terminal :facepalm:, now the game does load! 

It seems the other graphical errors have to do with improper scaling, which I'm not sure how to fix, I'd assume there's some sort of setting I'm missing?

Correct on Epiphany:
![Screenshot_2024-01-23_10:23:02](https://github.com/tauri-apps/tauri/assets/20760920/b1806da1-9f83-4ad3-aa51-b1295f080fc6)
![Screenshot_2024-01-23_10:23:27](https://github.com/tauri-apps/tauri/assets/20760920/f1504b65-8333-46bf-8778-a14c6dcce5a0)

Incorrect on Tauri:
![Screenshot_2024-01-23_10:23:07](https://github.com/tauri-apps/tauri/assets/20760920/12d3fd41-177b-4737-9130-37b5af26cc1c)
![Screenshot_2024-01-23_10:23:24](https://github.com/tauri-apps/tauri/assets/20760920/fa97fa6f-5913-40bd-aa42-6bdc76cc1166)

**Reply by FabianLars** [reply]
improper scaling is a known issue (https://github.com/tauri-apps/tauri/issues/8602 for example). i don't know how to fix that yet though.

**Reply by huantianad** [reply]
Interesting, I also did see these bug reports about scaling. However, in those cases I usually saw that Epiphany also had those scaling issues too! So when I saw that I didn't get those issues in epiphany, I wondered if I missed something on the tauri side.
